clients/src/test/java/org/apache/kafka/common/record/EosLogRecordTest.java

```diff
@@ -27,94 +27,48 @@
 
     @Test
     public void testBasicSerde() {
-        ByteBuffer key = ByteBuffer.wrap("hi".getBytes());
-        ByteBuffer value = ByteBuffer.wrap("there".getBytes());
-        int delta = 57;
-        long timestamp = System.currentTimeMillis();
-
-        ByteBuffer buffer = ByteBuffer.allocate(1024);
-        EosLogRecord.writeTo(buffer, delta, timestamp, key, value);
-        buffer.flip();
-
-        EosLogRecord record = EosLogRecord.readFrom(buffer, 1L, 0, null);
-        assertNotNull(record);
-        assertEquals(0, record.attributes());
-        assertEquals(58L, record.offset());
-        assertEquals(57, record.sequence());
-        assertEquals(timestamp, record.timestamp());
-        assertEquals(key, record.key());
-        assertEquals(value, record.value());
-    }
-
-    @Test
-    public void testSerdeNullKey() {
```

Comment:
They are now covered in `testBasicSerde`. I thought the redundancy was a little annoying.